document_ftp/ftpserver/abstracted_fs.py

```diff
@@ -0,0 +1,667 @@
+# -*- encoding: utf-8 -*-
+
```

Comment:
Sorry for not noticing that. Then Openerp SA copyright should be left in the code files and in the manifest's list of authors.
A PEP8/style cleanup should be done, nevertheless.